src/generate/templates/ping-message-all.h.in

```diff
@@ -11,11 +11,38 @@
 #include "ping-message-{{definition}}.h"
 {% endfor %}
 
+// Declare an empty Q_NAMESPACE and Q_ENUM_NS if it's not a Qt project, otherwise include QObject
+#ifndef QT_CORE_LIB
+#define Q_NAMESPACE
+#define Q_ENUM_NS(x) //x
```

Comment:
`//x` ?

Reply:
This is just something to make it explicit that such variable is not used, but if it's make more confuse, I'll remove it.